src/data/pokemon/item_effects.h

```diff
@@ -325,42 +325,66 @@ const u8 gItemEffect_DireHit[8] = {
 };
 
 const u8 gItemEffect_XAttack[8] = {
+#ifdef BATTLE_ENGINE
+    [1] = ITEM1_X_ATTACK,
+#else
     [0] = 1,
+#endif
     [5] = ITEM5_FRIENDSHIP_LOW | ITEM5_FRIENDSHIP_MID,
     [6] = 1,
     [7] = 1,
 };
 
 const u8 gItemEffect_XDefend[8] = {
```

Comment:
```suggestion
const u8 gItemEffect_XDefense[8] = {
```
Because the name was changed in newer Gens.

Reply:
I think renames to items should all be handled in a separate PR, personally.

There's a lot of those to do due to the many changes in item's names introduced in Gen. 8 anyway.

It feels like it'd be more effective to get them all out of the way in one single PR, but if that's not good, I'll apply this change :eyes:

Reply:
It appears the actual item rename has partially been done already, but a few things were missed which a different PR is covering already. Therefore I suggest we cover the item effect related stuff (like the rework defines that you newly added in the BE PR) in these PRs. I don't think there's a point in adding a define knowing we'll change it later, you know?

Reply:
What do you mean by "adding a define knowing we'll change it later"? :eyes:

Reply:
Well it's not exactly that in the case of `ITEM1_X_DEFEND` (although you are assigning a different value to it so technically it's a new define?), but you are adding some new defines like `ITEM1_X_ATTACK`, etc. (some stats were labelled differently because the used different flag sets in the old code). So basically since `ITEM1_X_DEFEND` is a different and new define, might as well rename it at the same time.